packages/html/ds/src/card/card.stories.ts

```diff
@@ -70,31 +79,44 @@ export const Default: Story = {
         type: { summary: 'string' },
       },
     },
-    actions: {
+    tag: {
       control: 'object',
-      description:
-        'Array of actions that are displayed as links at the bottom of the card.',
+      description: 'Tag displayed on the card.',
+      table: {
+        category: 'Card Content',
+        type: { summary: 'Tag' },
+      },
+    },
+    action: {
+      control: 'object',
+      description: 'Single action button or link for the card.',
       table: {
         category: 'Actions',
-        type: { summary: 'Array<{ href: string, text: string }>' },
+        type: { summary: '{ type: "link" | Button | Link }' },
```

Comment:
is this right ? shouldn't it be `{ summary: '{ type: | Button | Link }' }` ?

Reply:
I should rephrase it a bit better because action is going to have `type` property that defines link or button.

Let me know what you think of this?|

E.g.

```
action: {
  control: 'object',
  description: 'Single action for the card, either a button or link, determined by the "type" property.',
  table: {
    category: 'Actions',
    type: { summary: '{ type: "button" | "link", ...Button | Link }' },
  },
},
```

Reply:
yeah, I think that works